tutorials/RandomNumberGeneration/RandomNumberGenerationTutorial.ipynb

```diff
@@ -191,9 +191,13 @@
     "In exercise 3, we generated numbers in the range $[0, 2^N-1]$ $(1 \\leq N \\leq 10)$. Now let's create an operation that will return a random number in the range $[min, max]$. \n",
     "\n",
     "**Input:** \n",
-    "Two integers $min$ and $max$ ($1 \\leq min \\leq max \\leq 2^{10}-1$).\n",
+    "A $range$ tuple of the form ($min$, $max$) such that ($0 \\leq min \\leq max \\leq 2^{10}-1$).\n",
```

Comment:
We don't need to change the signature here either.

Reply:
Done